qiskit_experiments/framework/restless_mixin.py

```diff
@@ -91,16 +95,23 @@ def enable_restless(
         # The excited state promotion readout analysis option is set to
         # False because it is not compatible with a restless experiment.
         if self._t1_check(rep_delay):
+            meas_level = self._default_run_options().get("meas_level", MeasLevel.CLASSIFIED)
+            meas_return = self._default_run_options().get("meas_return", MeasReturnType.AVERAGE)
```

Comment:
I think that this should always be set to single-shot for restless.

Reply:
Implemented here: [315b93a](https://github.com/Qiskit/qiskit-experiments/pull/752/commits/315b93a02555acc1352497c0edb0994d63c419e2).